laletterbuilder/schema.py

```diff
@@ -220,10 +226,63 @@ def perform_mutate(cls, form, info: ResolveInfo):
         return cls.mutation_success()
 
 
+@schema_registry.register_mutation
+class LaLetterBuilderSendOptions(SessionFormMutation):
+    class Meta:
+        form_class = SendOptionsForm
+
+    login_required = True
+
+    @classmethod
+    def resolve(cls, parent, info: ResolveInfo):
+        return super().resolve(parent, info)
+
+    @classmethod
+    @mutation_requires_onboarding
+    def perform_mutate(cls, form, info: ResolveInfo):
+        # Get most recent unsent letter. This relies on there
+        # only being one unsent habitability letter at a time.
+        # TODO: copied from LaLetterBuilderIssues, refactor to shared function in session
```

Comment:
FYI: As it is, the following lines aren't tested in `test_schema.py` which is OK for now, but once you refactor it you'll find the tests for these lines in `TestLaLetterBuilderIssuesMutation`: `test_it_raises_err_when_no_letter_exists` and `test_it_raises_err_when_multiple_unsent_letter_exists`.